contents/feedback/progress/quest-progress/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { CircleCheckBigIcon } from "@yamada-ui/lucide"
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  Center,
+  Container,
+  HStack,
+  Heading,
+  Progress,
+  Text,
+  VStack,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+
+const mockDatas = [
+  {
+    content: "Solve 5 question",
+    current: 5,
+    max: 5,
+  },
+  {
+    content: "Solve 10 question",
+    current: 10,
+    max: 10,
+  },
+  {
+    content: "Solve 15 question",
+    current: 12,
+    max: 15,
+  },
+]
+
+const QuestProgress: FC = () => {
+  return (
+    <Container m="auto">
+      <Heading>Daily Quests</Heading>
```

Comment:
What's this `m="auto"` for?

Reply:
Also, a card variant of `outline` or `solid` may look better.

Reply:
> What's this `m="auto"` for?

I just forgot to delete it.

Reply:
> Also, a card variant of `outline` or `solid` may look better.

Can you give us an example?

Reply:
> > Also, a card variant of `outline` or `solid` may look better.
> 
> Can you give us an example?

@illionillion They would look like below.
![image](https://github.com/yamada-ui/yamada-components/assets/61367823/a208ccc3-b2df-43d0-8a6f-24d5c0456c2e)

```ts
<Card flexDir="row" key={`${data.content}-${i}`} variant="solid">
```
or
```ts
<Card flexDir="row" key={`${data.content}-${i}`} variant="outline">
```

Reply:
I see.
Thanks!!

Reply:
I fixed.